src/button.tsx

```diff
@@ -79,6 +79,15 @@ const renderButtonElement = ({
     return resultChildrenArr;
 };
 
+const ButtonLinkChevron: React.FC = () => (
```

Comment:
why didn't you include this icon as a custom icon in the `src / icons` folder ?

Reply:
Discussed offline and agreed on leaving it here. This icon is specific for buttons. We don't want other components to be able to use it.